helm-charts/splunk-otel-collector/templates/daemonset.yaml

```diff
@@ -516,7 +512,7 @@ spec:
       {{- if .Values.isWindows }}
       - name: hostfs
         hostPath:
-          path: "C:\\"
```

Comment:
Q - was the 2nd backslash meant for escaping purposes?

Reply:
Yes, inside double-quoted strings in YAML one needs to escape the `\`, remove the double-quoting and no escape is needed.

Reply:
Rule of thumb for Windows file paths in YAML files:
```YAML
exporters:
  file/path-backslash:
    path: c:\Users\pjanotti\src\spikes\dir with spaces\logs\backslash.log
  file/path-forwardslash:
    path: c:/Users/pjanotti/src/spikes/dir with spaces/logs/forwardslash.log
  file/path-backslash-nodrive:
    path: \Users\pjanotti\src\spikes\dir with spaces\logs\backslash-nodrive.log
  file/path-forwardslash-nodrive:
    path: /Users/pjanotti/src/spikes/dir with spaces/logs/forwardslash-nodrive.log
  file/path-with-quotes: # needs to escape '\'
    path: "c:\\Users\\pjanotti\\src\\spikes\\dir with spaces\\logs\\with quotes.log"
```